worker/src/RTC/Transport.cpp

```diff
@@ -34,6 +35,34 @@ namespace RTC
 	{
 		MS_TRACE();
 
+		auto jsonDirectIt = data.find("direct");
+
+		// clang-format off
+		if (
+			jsonDirectIt != data.end() &&
+			jsonDirectIt->is_boolean() &&
+			jsonDirectIt->get<bool>()
```

Comment:
isn't is redundant?

Reply:
Because we want to check if it's value is true, not just if it's  a boolean.

Reply:
right  :+1: